exporters/otlp/otlp.go

```diff
@@ -231,3 +239,51 @@ func (e *Exporter) uploadTraces(ctx context.Context, sdl []*tracesdk.SpanData) e
 	}
 	return err
 }
+
+func NewExportPipeline(ctx context.Context, exportOpts []ExporterOption,
+	resOpts []resource.Option) (*Exporter, *sdktrace.TracerProvider, *push.Controller, error) {
+
+	exp, err := NewExporter(ctx, exportOpts...)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	res, err := resource.New(ctx, resOpts...)
```

Comment:
None of the examples from the tracking issue allow passing resource options.  Why are we adding them here?

Reply:
Actually i take the default behaviour from this PR https://github.com/open-telemetry/opentelemetry-collector/pull/2159 and this PR is an origin of the issue, so i decided to add resource as well.

Reply:
All the `New` convenience functions in this repository are missing resources if any of them are. I suggest we think about either making resource options a subset of all the options used in the various `InstallNewPipeline` methods through this repository (i.e., make there be one list of options, not two lists). An example of this kind of option support is in the `trace` package, see `LifeCycleOption`, `SpanOption`, and `EventOption`.

An alternative might be to have a global set of resource detectors and just make sure a default is used that most users will be happen with, so that we don't have to add resource support in every one of the convenience functions.

Reply:
I see that you removed the resource options from this PR. Thanks.

I think it would make sense to file an issue about adding resource support consistently for all the "installers". I'll do that.

Reply:
https://github.com/open-telemetry/opentelemetry-go/issues/1391